Podcast/RecommendedSeriesTableViewCell.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  RecommendedSeriesTableViewCell.swift
+//  Podcast
+//
+//  Created by Kevin Greer on 2/19/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+protocol RecommendedSeriesTableViewCellDataSource {
+    func recommendedSeriesTableViewCell(dataForItemAt indexPath: IndexPath) -> Series
+    func numberOfRecommendedSeries() -> Int
```

Comment:
how does the vc implementing this function know which cell it's providing the number of recommended series for?
should be func numberOfRecommendedSeries(forRecommendedSeriesTableViewCell cell: RecommendedSeriesTableViewCell) -> Int